src/openassetio-core/include/openassetio/hostApi/Manager.hpp

```diff
@@ -607,6 +636,10 @@ class OPENASSETIO_CORE_EXPORT Manager {
    * the Manager may even use this opportunity to switch to some
    * temporary working path or some such.
    *
+   * @warning If the supplied @ref trait_set is missing traits required
+   * by the manager for any input entity reference, then that element
+   * will error.
```

Comment:
[Consider] Could we state the specific error type that will be caused in this case?

Reply:
Oh darn, no, not just to satisfy a doc change. We should maybe [plan] to add one though.